src/formatDate.js

```diff
@@ -8,7 +8,7 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  return null;
 }
```

Comment:
The body of the `formatDate` function is currently returning `null`, which is a placeholder. To adhere to the task description, you need to implement the logic that converts the input `date` from the `fromFormat` to the `toFormat`. The function should parse the date according to the `fromFormat`, then reformat and return it according to the `toFormat`.